### PR TITLE
Skybrowser drag and drop for reordering images

### PR DESCRIPTION
**Pull request by ylvaselling**
Change the reordering image layer interface to drag and drop. 

Use with https://github.com/OpenSpace/OpenSpace/pull/2321

### CODE REVIEW

File: src/components/BottomBar/SkyBrowser/SkyBrowserEntry.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    font-size: 16px;
+  }
+}
+
+.sliderContainer {
+  width: 160px; 
+}
```
Comment (WeirdRubberDuck) [inline src/components/BottomBar/SkyBrowser/SkyBrowserEntry.scss:81]: Missing newline at end of file

File: src/components/BottomBar/SkyBrowser/SkyBrowserFocusEntry.jsx [hunk 2]
```diff
@@ -46,58 +16,29 @@ function SkyBrowserFocusEntry({
...
-          </Button>
-        </div>
-        )}
         <div className={styles.image}>
-          <LazyLoadImage src={thumbnail} alt={''} className={styles.imageText} onClick={setOpacity ? select : undefined}/>
+          <LazyLoadImage src={thumbnail} alt={''} className={styles.imageText} />
         </div>
         <div className={styles.imageHeader}>
-          <span className={styles.imageTitle}>
+          <span className={styles.imageTitle} style={{  width: '80px' }}>
```
Comment (WeirdRubberDuck) [inline src/components/BottomBar/SkyBrowser/SkyBrowserFocusEntry.jsx:41]: nitpicky, but two spaces after `{{`

File: src/components/BottomBar/SkyBrowser/SkyBrowserTooltip.scss [hunk 3]
```diff
@@ -78,3 +78,10 @@ $tooltip-color: rgba(15,15,15,0.9); //rgba(60,60,60,0.95);
   top: -2*$arrow-size;
   left: 88px;
 }
+
+.left::after {
+  border-bottom-color: $tooltip-color; //rgba(255,255,255,0.3);
+  top: 22px;
+  left: 123px;
+  transform: rotate(90deg);
+}
```
Comment (WeirdRubberDuck) [inline src/components/BottomBar/SkyBrowser/SkyBrowserTooltip.scss:87]: odd, but this is apprently no new line?

**Review by WeirdRubberDuck — APPROVED** [review]
Looks great to me!  :) 

Something that was really noticable when trying this out though was the lack of performance for the CEF gui... See video. The delay when dragging until the movement happens if very apparent. We don't have the same problem when opening the UI in a browser

https://user-images.githubusercontent.com/8808894/202706362-3f8c2c27-c5b7-4d35-a1f1-195dc04cdb26.mov

But we already have an issue for this, right?

**Comment by ylvaselling** [comment]
I've updated the PR with some commits to address the performance issue. Hopefully it will make a difference. I never got the problem to begin with so I would be grateful for some testing 🙏

**Comment by alexanderbock** [comment]
Can confirm that the performance is good now :+1:

**Comment by WeirdRubberDuck** [comment]
A lot better indeed!

**Pull request merged**